packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.routing.js

```diff
@@ -3,6 +3,13 @@ export default /* @ngInject */ ($stateProvider) => {
     .state('pci.projects.project.storages.blocks', {
       url: '/blocks?help',
       component: 'pciProjectStorageBlocks',
+
+      redirectTo: (trans) => {
+        const resolvePromise = trans.injector().getAsync('storages');
```

Comment:
Any reason not to 
```suggestion
        return trans.injector()
          .getAsync('storages')
          .then(...)
```
?